ui/src/common/components/NewFeatureTag/NewFeatureTag.jsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
 import { Tag } from 'antd';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import styleVariables from '../../../styleVariables';
 
 import './NewFeatureTag.scss';
 
 const GREEN = styleVariables['success-color'];
 
-function NewFeatureTag() {
+function NewFeatureTag({ className }) {
   return (
-    <Tag className="__NewFeatureTag__" color={GREEN}>
+    <Tag className={classnames('__NewFeatureTag__', className)} color={GREEN}>
       New
     </Tag>
   );
 }
 
+NewFeatureTag.propTypes = {
+  className: PropTypes.string,
```

Comment:
although there is not a lot of value in that, but you can create a new test now like "renders with class name" and snapshot `<NewFeatureTag className="test"...`